modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
K, your 'package_name' was not displayed -- now I can follow.
I am agree then with having that as a default and not having a flag, however this would still be a breaking change for libraries with extensions since you have to adapt the require statements?